S/Sundials/Sundials@5/build_tarballs.jl

```diff
@@ -6,14 +6,33 @@ version = v"5.2.1" # <-- There is no version 5.2.1, but we need to change versio
 # Collection of sources required to build Sundials
 sources = [
     GitSource("https://github.com/LLNL/sundials.git",
-              "b16d3d3995668c9a13c9f4bee8b0113ff6a9cf6d"),
+              "8264ba5614fd9578786e5e8a3ba9f703ff795361"),
```

Comment:
This is because the commit sha that was here before is not in the upstream repo anymore - not sure what they did with it, but it used to point to the 5.2.0 commit and they do have a tag for that, so use that instead.